Framework/PythonInterface/plugins/algorithms/HB3AAdjustSampleNorm.py

```diff
@@ -0,0 +1,272 @@
+
+        # Alternative WS inputs
+        self.declareProperty("InputWorkspaces", defaultValue="", direction=Direction.Input,
+                             doc="Workspace or comma-separated workspace list containing input MDHisto scan data.")
+        self.declareProperty(IMDHistoWorkspaceProperty("VanadiumWorkspace", defaultValue="", direction=Direction.Input,
+                                                       optional=PropertyMode.Optional),
+                             doc="MDHisto workspace containing vanadium normalization data")
+
+        # Detector adjustment options
+        self.declareProperty("DetectorHeightOffset", defaultValue=0.0, direction=Direction.Input,
+                             doc="Optional distance to move detector height (relative to current position)")
+        self.declareProperty("DetectorDistanceOffset", defaultValue=0.0, direction=Direction.Input,
+                             doc="Optional distance to move detector distance (relative to current position)")
+
+        self.declareProperty("Wavelength", defaultValue="",
+                             doc="Optional wavelength value to use as backup if one was not found in the sample log")
+
+        # Which conversion algorithm to use
+        self.declareProperty("OutputAsMDEventWorkspace", defaultValue=True, direction=Direction.Input,
+                             doc="Whether to use ConvertHFIRSCDtoQ for an MDEvent, or ConvertWANDSCDtoQ for an MDHisto")
+
+        # MDEvent WS Specific options for ConvertHFIRSCDtoQ
+        self.declareProperty(FloatArrayProperty("MinValues", [-10, -10, -10], FloatArrayLengthValidator(3),
+                                                direction=Direction.Input),
+                             doc="3 comma separated values, one for each q_sample dimension")
+        self.declareProperty(FloatArrayProperty("MaxValues", [10, 10, 10], FloatArrayLengthValidator(3),
+                                                direction=Direction.Input),
+                             doc="3 comma separated values, one for each q_sample dimension; must be larger than"
+                                 "those specified in MinValues")
+        self.declareProperty("MergeInputs", defaultValue=False, direction=Direction.Input,
+                             doc="If all inputs should be merged into one MDEvent output workspace")
+
+        # MDHisto WS Specific options for ConvertWANDSCDtoQ
+        self.declareProperty(FloatArrayProperty("BinningDim0", [-8.02, 8.02, 401], FloatArrayLengthValidator(3),
+                                                direction=Direction.Input),
+                             "Binning parameters for the 0th dimension. Enter it as a"
+                             "comma-separated list of values with the"
+                             "format: 'minimum,maximum,number_of_bins'.")
+        self.declareProperty(FloatArrayProperty("BinningDim1", [-0.82, 0.82, 41], FloatArrayLengthValidator(3),
```

Comment:
Defaults for BinningDim0 and BinningDim2 are good. I suggest changing BinningDim1 to `[-2.52, 2.52, 126]`